pom.xml

```diff
@@ -465,7 +466,16 @@
         <artifactId>license-maven-plugin</artifactId>
         <configuration>
           <skip>${skipLicense}</skip>
-          <inlineHeader>
+          <properties>
+            <owner>Dremio</owner>
+            <project.inceptionYear>2020</project.inceptionYear>
```

Comment:
nit: if we set `<inceptionYear>` on the module pom, we dont have to repeat it here, it can then be used as `${project.inceptionYear}` or `${year}` in the license